modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -3988,7 +3986,7 @@ m|TOPOLOGY
 m|03N85
 |Status description
 a|
-info: server has caught up. Server `${ name }` at address `${ address }` has caught up.
+info: server has caught up. Server `{ $serverName }` at address `{ <<serverAddress>> }` has caught up.
```

Comment:
```suggestion
info: server has caught up. Server `{ <<serverName>> }` at address `{ <<serverAddress>> }` has caught up.
```

It should be like this right? That are also some other instances of this below. Or was it intentional because we do not have a description of serverName and want to change it to server instead? 🤔

Reply:
It's intentional. We don't have `{ $serverName }`, we have `{ $server }` instead. When it's changed in the code, we'll change it here as well. Otherwise, we'll end up with both having the same description.

Reply:
Ok that is what I remembered after starting to write this comment